xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/extension/extension.css

```
@@ -475,3 +476,21 @@ ul.collapsible.document-tree {
   font-weight: normal;
   padding-right: .3em;
 }
+
+/* Media query for screens wider than Bootstrap @screen-md-min */
+@media (min-width: 992px) {
+  /* Display the extension title and status on a single row on md screens */
+  .extension-title, .dependency-item {
+    display: flex;
```

```suggestion
    display: flex;
    align-items: center;
```

I think you're just missing the vertical alignment.

---

Indeed thx